build/templates/build-number.yml

```diff
@@ -9,3 +9,31 @@ steps:
       $buildNumberWithPadding = [int]$(PreReleaseNumber) + [int]$(BuildPadding)
       Write-Host "##vso[task.setvariable variable=BuildNumber]$buildNumberWithPadding"
       Write-Host "Overridden build number: $buildNumberWithPadding"
+
+# For all builds, we multiply the actual build number by 10 to act as a shift to the left.
```

Comment:
Is this required? It's probably only for internal applications.

Reply:
This was part of an experiment that seems to not have worked. Removed.